starter-code/index.html

```diff
@@ -0,0 +1,206 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="css/bootstrap.min.css">
+
+    <!-- My Own CSS -->
+    <link rel="stylesheet" href="css/styles.css">
+
+    <title>clone - revera</title>
+  </head>
+
+  <!-- top header -->
+  <div class="container-fluid my-nav-container">
+    <nav id="my-navigation" class="navbar navbar-expand-lg navbar-light bg-light">
+      <div class="space col-md-1"></div>
+      <h1><a id="my-company-name" class="navbar-brand" href="#">REVERA</a></h1>
+      <div class="space col-md-3"></div>
+      <div class="collapse navbar-collapse" id="navbarNavDropdown">
+        <ul class="navbar-nav">
```

Comment:
nav links should be centered a bit more. you can always adjust width of the div above this one.